src/test/scala/chiselTests/BoringUtilsTapSpec.scala

```diff
@@ -228,4 +228,68 @@ class BoringUtilsTapSpec extends ChiselFlatSpec with ChiselRunners with Utils wi
     e.getMessage should include("Cannot drill writable probes upwards.")
   }
 
+  it should "work when tapping an element within a Bundle" in {
+    val chirrtl = circt.stage.ChiselStage.emitCHIRRTL(
+      new RawModule {
+        class MiniBundle extends Bundle {
+          val x = Bool()
+        }
+        class Child() extends RawModule {
+          val b = Wire(new MiniBundle)
+        }
+
+        val child = Module(new Child())
+
+        // directly tap Bundle element
+        val outRWProbe = IO(probe.RWProbe(Bool()))
+        probe.define(outRWProbe, BoringUtils.rwTap(child.b.x))
+
+        // tap Bundle, then access element
+        val outRWBundleProbe = IO(probe.RWProbe(new MiniBundle))
+        val outElem = IO(probe.RWProbe(Bool()))
+        probe.define(outRWBundleProbe, BoringUtils.rwTap(child.b))
+        probe.define(outElem, outRWBundleProbe.x)
+      }
+    )
+    matchesAndOmits(chirrtl)(
+      "wire b : { x : UInt<1>}",
+      "define bore = rwprobe(b.x)",
+      "define bore_1 = rwprobe(b)",
```

Comment:
I don't love the reliance on a default name coming from BoringUtils, is the use of BoringUtils really necessary or can we test the same behavior by just having the probe IOs on Child?

Reply:
Or put another way, are we trying to test the behavior of BoringUtils here or just basic tapping of aggregate children? It appears to me it's more about the latter and the use of BoringUtils is just making the test a little more concise?

Reply:
> are we trying to test the behavior of BoringUtils here or just basic tapping of aggregate children?

Kind of both... the new tap API is implemented in BoringUtils and relies on bores, which is why bores are being created.

Reply:
I think I missed that this test is in the file `BoringUtilsTapSpec`, if that's what we're testing, that's what we should be testing 😅